src/OnlineSales/Services/VariablesService.cs

```diff
@@ -16,6 +16,18 @@ public VariablesService(IEnumerable<IVariablesProvider> variableProviders)
         this.variableProviders = variableProviders;
     }
 
+    public string[] GetProjectVersionsList(string project, string version, string language)
```

Comment:
@Mixer57 VariablesService is a generic service which has very simple idea - it should provide any sort of variables without knowing what these variables are. Here you introduced very specific logic with projects and versions. VariablesService is not a good place for that kind of logic.

Reply:
Removed and rewrite logic